sc-memory/sc-memory/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ add_library (sc-memory SHARED ${SOURCES} ${SOURCES_PARSER})
 target_include_directories(sc-memory
       PRIVATE ${GLIB2_INCLUDE_DIRS}
       PRIVATE ${SC_MACHINE_ANTRL_DIR}/runtime/src
-      PRIVATE ${SC_MEMORY_SRC}
+      PUBLIC ${SC_MEMORY_SRC}
```

Comment:
Same as previous. This is a part of this PR, because it is required to implement benchmarks.
We don't need to add these changes to changelog because they don't change anything for a customers.

Reply:
I already wrote: this change is important for customers because sc-memory target users had to do include_directories(${SC_MEMORY_SRC}). After this change they don't need to do it anymore

Reply:
Create issue for using SC_MEMORY_SRC and resolve it for release 0.7.0